### DISCUSSION TITLE
0.67.* notification of modified document from people not allowed to do so

### DISCUSSION THREAD

**Discussion by bmm-alc** [outline/outline#4726] [Self Hosting] [answered]
Hello,

Since 0.67, some users reported they received mail notification of modified document from people they're are not allowed to do so. Although I can see a revision from this person in the document history.

Did someone get the same bug ? How can I investigate this ?

It's hard to spot differences between 2 revisions in the UI, is there a way to see a diff-like format (with some metadata)

Thanks

**Comment by tommoor** [comment, marked as answer]
This should be addressed by 0cccd7141dac92f4b0a45ce6107a8609ffcf5534 once it makes it into a release. It's happening due to a fix in the Markdown written into the db, opening the document causes the fixed Markdown to get written and the doc "changed".

**Reply by bmm-alc** [reply]
Hi Tom,

thanks for info, I'll inform my users.

Best.

**Reply by bmm-alc** [reply]
thanks for doing release 0.67.2 with the fix
